pkg/ddevapp/webserver_config_assets/nginx-site-magento2.conf

```diff
@@ -1,4 +1,5 @@
 # ddev magento2 config
+# https://github.com/magento/magento2/blob/2.4-develop/nginx.conf.sample
```

Comment:
I'm amazed that we didn't have to change things here!

Reply:
It does look different, but I only deleted our media stanza, thanks for noticing this.